### PR TITLE
RWIB Mission 2 Community Page

### PR DESCRIPTION
**Pull request by efindlay**
- Removed links on front page, and replaced with 'news', 'community' and 'legal'
- Major changes to Community page
- Added User_Group page

### CODE REVIEW

**Comment by brson** [comment]
Thanks @efindlay. I'll review Tuesday.

**Comment by brson** [comment]
I've pushed PRS #201 (community), #200 (docs), and #199 (contributing) to [my own gh pages so reviewers can see](http://brson.github.io/rust-www/). The first goal is to get the community page merged since it's the most developed.

The key pages under review are [community.html](http://brson.github.io/rust-www/community.html), [user_groups.html](http://brson.github.io/rust-www/user_groups.html), [documentation.html](http://brson.github.io/rust-www/documentation.html), and [contributions.html](http://brson.github.io/rust-www/contributions.html)

**Comment by edunham** [comment]
Should the lists of IRC channels actually be bulleted lists? If they're paragraphs, the hyphens look out of place. (going from how it's rendered on http://brson.github.io/rust-www/community.html)

**Comment by brson** [comment]
Some of these comments are carried over from the previous PR.

"Rust has a lot going for it as a language, but its greatest strength is its tight-knit community and its focus on kindness and inclusivity." Can we simplify this to "Rust is an amazing language, but its greatest strength is its kind and inclusive community."

'We also have a twitter account' is terminated with a comma, should be a period.

The reformatting paragraphs of IRC channels contain inline dashes like a bulleted list. These are weird and I suggest removing them.

The stylistic difference between h2 and h3 is very subtle. I'd like to try to make them more distinct.

On the usergroups page, 'meetings are ... very informal'. Suggest removing 'very', filler word. Perhaps combine with the following sentence: 'meetings are usually informal and open to everyone', removing the 'monthly' assertion, since I'm not confident it generally holds.

On the usergroups page, 'rust_www' i'd change to either 'rust-www' to match the repo name, or maybe just 'the website'.

Try including the calendar again on the usergroups page, indicating that meetups are often listed there, perhaps even suggesting to contact the community team to add events.

I'll take a crack at some of these myself and send you a PR in a bit.

**Comment by brson** [comment]
@edunham The markup indicates that they are intended to be bulleted. I'll fix that and send to @efindlay .

**Comment by brson** [comment]
[Here's my PR against @efindlay's to address most of my above comments](https://github.com/efindlay/rust-www/pull/2).

**Comment by brson** [comment]
Once that PR is merged I'll post this to the wider community to get more feedback.

**Comment by brson** [comment]
I sent @efindlay another minor PR to fix a broken link: https://github.com/efindlay/rust-www/pull/3

File: community.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+- The [Users Forum][users_forum], a space for asking questions, posting code
+  snippets, talking about Rust projects, and so on.
+
+- The [Internals Forum][internals_forum], a space dedicated to design and
+  implementation discussion about Rust itself (which includes Cargo, the
+  standard library, and other core bits of infrastructure).
+
+[users_forum]: https://users.rust-lang.org/
+[internals_forum]: https://internals.rust-lang.org/
+
```
Comment (skade) [inline community.md]: Can we add local forums here? forum.rustplatz.de is a gathering for German-speaking people.

File: community.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+outreach, commercial users, teaching materials, and exposure.
+
+* The [Moderation Team][mod_team] is responsible for helping to uphold the
+[code of conduct][coc].
+
+In addition to the official team rosters, most teams also have a larger set
+of reviewers who are knowledgeable about the area and can sign off on code.
+
+If you're interested in getting involved in one of these teams, feel free to
+reach out to the team leader, who can help get you started.
```
Comment (skade) [inline community.md]: I think people can reach out to any member of the team, they can forward. Easier for all.

File: community.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+* The [Compiler Team][compiler_team] is responsible for compiler internals and
+optimizations.
+
+* The [Tooling and Infrastructure Team][tool_team] is responsible for official
+tools like Cargo, multirust, rustfmt, as well as the CI Infrastructure for the
+project.
+
+* The [Community Team][community_team] is responsible for coordinating events,
+outreach, commercial users, teaching materials, and exposure.
```
Comment (skade) [inline community.md]: The community team is also approachable for any question that might not have a clear target.

File: user_groups.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+[Lille Rust Meetup](http://www.meetup.com/rust-lille/), Lille.
+
+[Rust Paris](http://www.meetup.com/Rust-Paris/), Paris.
+
+### Germany
+
+[Rust Cologne/Bonn User Group](http://www.meetup.com/Rust-Cologne-Bonn/), Köln.
+
+[Rust Hack and Learn](http://www.meetup.com/Rust-Berlin/), Berlin.
```
Comment (skade) [inline user_groups.md]: It's actually "Rust Berlin". Rust Hack and learn, is just one of the meetups.

File: user_groups.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+### Australia
+
+[Rust Melbourne](http://www.meetup.com/Rust-Melbourne/), Melbourne.
+
+[Rust Sydney](http://www.meetup.com/Rust-Sydney/), Sydney.
+
+### Austria
+
+[Klagenfurt Rust Programmers](http://www.meetup.com/Klagenfurt-Rust/), Klagenfurt.
+
```
Comment (skade) [inline user_groups.md]: There is one in Sao Paolo, Brazil.

http://www.meetup.com/Rust-Sao-Paulo-Meetup/

**Comment by skade** [comment]
Nice! Made a run though and added some detail.

**Comment by brson** [comment]
@bstrie  suggests we use mibbit links instead of irc.

**Comment by efindlay** [comment]
> Can we add local forums here? forum.rustplatz.de is a gathering for German-speaking people.

@skade, @brson, Rather than adding a link to the German-speaking forum, I think we need to have a more general solution that encompasses all languages. What do you think about changing the 'User Groups' page to a 'International' page. That way we could make it the hub for not just user groups but also links to local forums and to translated Rust books. The 'International' link could go on the front page as well.

**Comment by skade** [comment]
@efindlay I'm surprised. The section says "Discussion Forums". Why would "International" be the appropriate section for this? I would like to avoid a "English/US" and "Everyone else" situation.

**Comment by efindlay** [comment]
@skade, I agree completely with avoiding an English/US and everyone else situation, and I'm certainly not dismissing the importance of the German-language forum and equally for any other language. While its no problem just to have a link to the German-language forum on the community page, it would be a problem to have 20 or so links to different language forums. But I think this problem of how to internationalize the website should be a put to the internals forum for open discussion, so how about I put the German-language forum link in as you suggested, at least for now, and try to collect ideas and opinions through the internals forum?

**Comment by skade** [comment]
Well, we have such a huge list of channels that I find ruling out a forum because of language very arbitrary.

As long as we don't fully internationalize the page (as the Ruby community does), I don't see how language should be a factor.

File: user_groups.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+[Rust Berlin](http://www.meetup.com/Rust-Berlin/), Berlin.
+
+[Rust Meetup Hamburg](http://www.meetup.com/Rust-Meetup-Hamburg/), Hamburg.
+
+[Rust Munich](http://www.meetup.com/rust-munich/), München.
+
+### India
+
+[Hyderabad Rust Meetup](http://www.meetup.com/Hyderabad-Rust-Meetup/), Hyderabad.
```
Comment (Manishearth) [inline user_groups.md]: Also, in Bangalore: https://www.facebook.com/groups/RustBLR/1579069959026339/

**Comment by brson** [comment]
@skade @efindlay OK. I'm going to merge this as-is, and follow up with another PR to tweak the wording about the community team per @skade. Punting on the German forum for the moment, but I want to make sure it gets surfaced somehow.

**Pull request merged**